lib/bkblz.rb

```diff
@@ -7,7 +7,7 @@ module Bkblz
 
   class << self
     def configure(&block)
-      @config = Bkblz::Config.configure @config, &block
+      @config = Bkblz::Config.configure (@config ||= nil), &block
```

Comment:
please include the &block inside the closing paren

also remove the whitespace before the opening paren